packages/amplify-cli/src/__tests__/plugin-helpers/verify-plugin.test.ts

```diff
@@ -48,12 +48,14 @@ describe('verify-plugin', () => {
     });
 
     it('returns PluginDirPathNotExist error when specify not exist path', async () => {
+      // eslint-disable-next-line @typescript-eslint/no-misused-promises
```

Comment:
these are false-positives in this file.
mocked function has non-promisified overloads that take callback and linter (or tsc?) is matching wrong one here.